### ISSUE TITLE
Google maps within a marker - Define parent feature request.

### ISSUE THREAD

**Issue by cboyce428** [fengyuanchen/viewerjs#37]
Firstly thanks for taking the time to put this together its great!.

If you try to use code within a google maps 'InfoWindow' although the viewerjs opens its not visible to the user (this can be proven it opens and loads by registering events).

This is due to the size of the parent tag from within the InfoWindow which is has a limited size. (the width of 'viewer-container' is zero so you see nothing)

Basically the hack to get it to work is changing:

_this.parent = parent = element.parentNode;

to become; 

_this.parent = parent = document.getElementById('photoGallery');

(The 'photoGallery' div is within the body of the page:). On a later version it would be good if you could specify the parent or it renders it at document level.

Chris

**Comment by fengyuanchen** [comment]
Just an image viewer, not work with other things.
